sdk/storage/Azure.Storage.Common/tests/Shared/StorageTestBase.cs

```diff
@@ -322,5 +324,14 @@ protected void AssertSecondaryStorage404OnSecondary(string primaryHost, string s
             }
             return response;
         }
+
+        internal void AssertResponseHeaders(TestConstants constants, SasQueryParameters sasQueryParameters)
```

Comment:
Any reason to take `TestConstants` here?

Reply:
This contains the expected values that we are asserting against.

Reply:
Oh, sorry.  Thought this was static but it's not.  Makes sense.